ckan/lib/helpers.py

```diff
@@ -50,18 +50,37 @@
 log = logging.getLogger(__name__)
 
 
+class HelperError(Exception):
+    """Raised if an attempt to access an undefined helper is made.
+
+    Normally, this would be a subclass of AttributeError, but Jinja2 will
+    catch and ignore them. We want this to be an explicit failure re #2908.
+    """
```

Comment:
@amercader @TkTech how do you feel about moving HelperError to ckan.exceptions? CKAN tends to have all its exceptions buried in the modules that use them, but in projects I've built recently I like to have my exceptions defined in a tiny shared module. It's helped me to prevent import loops and Exceptions feel like part of the public interface to the code.

At a minimum we should add it to ckan.plugins.toolkit so extensions can check for the existence of helpers in a safe way.